front/app/components/SignUpIn/AuthProviders.tsx

```diff
@@ -46,12 +46,13 @@ interface InputProps {
 }
 
 interface DataProps {
-  tenant: GetAppConfigurationChildProps;
-  passwordLoginEnabled: boolean | null;
-  googleLoginEnabled: boolean | null;
-  facebookLoginEnabled: boolean | null;
   azureAdLoginEnabled: boolean | null;
+  facebookLoginEnabled: boolean | null;
   franceconnectLoginEnabled: boolean | null;
+  googleLoginEnabled: boolean | null;
+  passwordLoginEnabled: boolean | null;
+  tenant: GetAppConfigurationChildProps;
+  viennaLoginEnabled: boolean | null;
```

Comment:
This is the new entry, the rest of the diff is coming from me sorting the items alphabetically :abcd:.

Reply:
Why would you sort it alphabetically?

Reply:
It's easier to find the item you look for next time.

Reply:
You can just use your editor's search, which will always be fastest, right?